test/components/entity/list.spec.js

```diff
@@ -188,20 +188,22 @@ describe('EntityList', () => {
       });
     });
 
-    it('does not show the modal during a refresh of the table', () => {
+    it('cancels a background refresh of the table', () => {
       testData.extendedEntities.createPast(1);
       return load('/projects/1/entity-lists/trees/entities', { root: false })
         .complete()
         .request(component =>
           component.get('#refresh-button').trigger('click'))
-        .beforeEachResponse(async (component) => {
-          await component.get('.entity-metadata-row .update-button').trigger('click');
-          component.getComponent(EntityUpdate).props().state.should.be.false;
+        .beforeAnyResponse(component =>
+          component.get('.entity-metadata-row .update-button').trigger('click'))
+        .respondWithData(() => {
+          testData.extendedEntities.createNew();
+          return testData.entityOData();
         })
-        .respondWithData(testData.entityOData)
         .respondWithData(testData.entityDeletedOData)
         .afterResponse(component => {
-          component.getComponent(EntityUpdate).props().state.should.be.false;
+          component.getComponent(EntityUpdate).props().state.should.be.true;
+          component.findAllComponents(EntityMetadataRow).length.should.equal(1);
```

Comment:
Add a comment, something like: "The new entity from the refresh response is not shown in the table."